Documentation/PLUGIN.md

```diff
@@ -83,6 +83,12 @@ $ protoc \
     --swift_out=. foo/bar/*.proto mumble/*.proto
 ```
 
+_NOTE:_ protoc 3.2.0 has an issue with the `--swift_opt` support. But there is
```

Comment:
Suggestion: "protoc 3.2.0 does not recognize `--swift_opt` if you rely on `protoc-gen-swift` being found on the `PATH`. To work around this, you need to explicitly add the argument..."